lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -194,6 +196,8 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
                 collection_method = CollectionMethod.KIOSK
             elif is_complete('test_order_survey', redcap_record_instance):
                 collection_method = CollectionMethod.SWAB_AND_SEND
```

Comment:
Ok, I was wondering about that overlap as well, since presence of a huksy_test_kit_registration on its own may not be a reliable enough way to determine if it's coming from a dropbox. Maybe it would be worth checking the barcode against ID3C to see if it's in the tiny swabs unobserved set.

Reply:
(Discussed on the call, just adding a note here for visibility) Unfortuantely, the barcode set would be the same for both of them too. I'm not sure we have a better way to distinguish other than the absense of the test order form, though I wish we did!